src/routes/namespaces/[namespace]/workflows/index.svelte

```diff
@@ -6,10 +6,11 @@
 
     const namespace = page.params.namespace;
     const timeRange = page.query.get('time-range');
+    const status = page.query.get('status') as WorkflowStatus;
```

Comment:
Should this as include null/undefined? in the case it doesn't exist in the query param? Or is it cool to pass that into the endpoints?

Reply:
`fetchClosedWorkflows` only adds it to the request if it's present since it's actively missing if we want everything.